src/Microsoft.Extensions.Configuration.Binder/project.json

```diff
@@ -6,7 +6,8 @@
     "nowarn": [ "CS1591" ],
     "xmlDoc": true
   },
-  "description": "Functionality to bind an object to data in configuration providers.",
+  "description": "Functionality to bind an object to data in configuration providers using the Microsoft.Extensions.Configuration.IConfiguration interface.",
```

Comment:
What about just "to data from configuration" or "to configuration settings". Not sure mentioning providers adds much value. I am on the fence on mentioning the interface.